src/components/common.js

```diff
@@ -290,15 +290,51 @@ const commonSelectProps = {
   },
   components: {
     Option: ({ children, ...props }) => {
-      return h(RSelectComponents.Option, props, [
+      return h(RSelectComponents.Option, {
+        ...props,
+        innerProps: _.merge(props.innerProps, {
+          role: 'option',
+          'aria-selected': props.isSelected
+        })
+      }, [
         div({ style: { display: 'flex', alignItems: 'center', minHeight: 25 } }, [
           div({ style: { flex: 1, minWidth: 0, overflowWrap: 'break-word' } }, [children]),
           props.isSelected && icon('check', { size: 14, style: { flex: 'none', marginLeft: '0.5rem', color: colors.dark(0.5) } })
         ])
       ])
+    },
+    SelectContainer: props => {
```

Comment:
For these, it might be tidier to destructure the argument:
```js
({ innerProps, ...props }) => {
  return RSelectComponents.SelectContainer({
    ...props
    innerProps: {
      ...innerProps,
      role: 'combobox',
      'aria-haspopup': 'listbox',
      'aria-expanded': isOpen
    }
  })
}
```

Alternatively, we could just merge once, since `_.merge` is a deep merge:

```js
return RSelectComponents.SelectContainer(_.merge(props, {
  innerProps: {
    role: 'combobox',
    'aria-haspopup': 'listbox',
    'aria-expanded': isOpen
  }
}))
```

Reply:
Thanks! `_.merge()` worked once I switched everything to using the `h` wrapper and pulling out the children property. But it's quite a bit cleaner now.